Items: on_wield and on_unwield callbacks implemented (#9862)
minetest/minetest: doc/lua_api.txt

        -- Shall drop item and return the leftover itemstack.
        -- The dropper may be any ObjectRef or nil.
        -- default: minetest.item_drop
        
        on_wield = function(itemstack, user),
        -- default: nil

>         -- It defines the behaviour of an item when it's wielded by a player.

Reviewer: ```suggestion
        -- Called when the player switches to this item in the hotbar.
```

Author: Done
